client/alias_options.go

```diff
@@ -0,0 +1,130 @@
+// Licensed to the LF AI & Data foundation under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import "github.com/milvus-io/milvus-proto/go-api/v2/milvuspb"
+
+// CreateCollectionOption is the interface builds CreateAliasRequest.
+type CreateAliasOption interface {
+	Request() *milvuspb.CreateAliasRequest
+}
+
+type createAliasOption struct {
+	collectionName string
+	aliasName      string
+}
+
+func (opt *createAliasOption) Request() *milvuspb.CreateAliasRequest {
+	return &milvuspb.CreateAliasRequest{
+		CollectionName: opt.collectionName,
+		Alias:          opt.aliasName,
+	}
+}
+
+func NewCreateAliasOption(collectionName, aliasName string) *createAliasOption {
```

Comment:
Maybe it would be better to change the parameter `aliasName` to `alias`. Keep it the same as func `NewDescribeAliasOption` and `NewDropAliasOption`, and keep it the same as pymilvus

Reply:
fixed